pytradfri/mood.py

```diff
@@ -1,22 +1,25 @@
 """Represent a mood on the gateway."""
+from __future__ import annotations
+
+from typing import cast
+
 from .const import ROOT_MOODS
-from .resource import ApiResource
+from .resource import ApiResource, TypeRaw
 
 
 class Mood(ApiResource):
     """Mood."""
 
-    def __init__(self, raw, parent):
+    def __init__(self, raw: TypeRaw, parent: str) -> None:
         """Create object of class."""
         super().__init__(raw)
         self._parent = parent
 
     @property
-    def path(self):
+    def path(self) -> list[str]:
         """Path."""
-        return [ROOT_MOODS, self._parent, self.id]
+        return [ROOT_MOODS, self._parent, cast(str, self.id)]
```

Comment:
The `id` is typed as an integer. So we need to copy to string if we want a string. We can only use `cast` when we want to tell mypy what type something is. It doesn't change the type.
```suggestion
        return [ROOT_MOODS, self._parent, str(self.id)]
```

Reply:
OK, thanks!